### PR TITLE
Add new style for navbar

### PR DESCRIPTION
**Pull request by AdelinaMoroaca**

### CODE REVIEW

**Review by talidag — CHANGES_REQUESTED** [review]

File: style/style.css [hunk 1]
```diff
@@ -343,6 +346,55 @@ header {
   background-color: #2f7973;
 }
 
+@media screen and (min-width: 320px) and (max-width: 768px) {
+  .nav__links__list {
+    display: none;
+    height: calc(100vh - 60px);
+    padding-top: 10rem;
```
Comment (talidag) [inline style/style.css:353]: Please remove this padding top.

File: style/scss/_header.scss [hunk 2]
```diff
@@ -375,5 +381,64 @@ header {
...
+@media screen and (min-width: $phone-size) and (max-width: $tablet-size) {
+    .nav {
+        &__links__list {
+            display: none;
+            height: calc(100vh - 60px);
+            padding-top: 10rem;
+            gap: 2rem;
+
+            & li > a{
+             font-size: 3rem;
```
Comment (talidag) [inline style/scss/_header.scss:393]: Try to use 24px here, instead of 30.

File: style/scss/_header.scss [hunk 3]
```diff
@@ -375,5 +381,64 @@ header {
...
+             font-weight: 500;
+            }
+    
+            & #socialLinks{
+                display: grid;
+                grid-template-columns: repeat(3, 1fr);
+                row-gap: 0.6rem;
+                column-gap: 1rem;
```
Comment (talidag) [inline style/scss/_header.scss:401]: Can you please try to make the gap between the icons bigger? Something like 2rem for rows and 3rem for columns.

File: style/scss/_header.scss (hunk 2 above)
Comment (AdelinaMoroaca) [reply style/scss/_header.scss:393]: added font-size: 2.4rem;

File: style/style.css (hunk 1 above)
Comment (AdelinaMoroaca) [reply style/style.css:353]: Padding top removed

File: style/scss/_header.scss [hunk 4]
```diff
@@ -375,5 +380,52 @@ header {
...
+            height: calc(100vh - 60px);
+            padding-top: 14rem;
+            gap: 1rem;
+
+            & li > a{
+             font-size: 2rem;
+             font-weight: 500;
+            }
+    
+            & #socialLinks{
```
[the remark this answers is not in the document]
Comment (AdelinaMoroaca) [reply style/scss/_header.scss:397]: Added new style : icons are displayed on 2 rows

File: style/scss/_header.scss (hunk 3 above)
Comment (AdelinaMoroaca) [reply style/scss/_header.scss:401]: Added   row-gap: 2rem; column-gap: 3rem;

File: script.js [hunk 5]
```diff
@@ -123,9 +123,65 @@ function hideMenuContent() {
...
-document
-  .querySelector(".close-menu-icon")
-  .addEventListener("click", hideMenuContent);
+
+document.querySelector('.close-menu-icon').addEventListener('click', hideMenuContent);
+
+
+
+function createNewSubList() {
+  const newList = document.createElement('ul');
```
[the remark this answers is not in the document]
Comment (AdelinaMoroaca) [reply script.js:132]: Added background-color: $white-smoke;  for icons and for `&__links__list` ( @media screen and (min-width: $phone-size) and (max-width: $tablet-size) ) too

File: style/scss/_header.scss [hunk 6]
```diff
@@ -375,5 +380,52 @@ header {
...
 }
 
-@media only screen and (max-width: 1025px) {
+@media screen and (min-width: $phone-size) and (max-width: $tablet-size) {
+    .nav {
+        &__links__list {
+            display: none;
+            height: calc(100vh - 60px);
+            padding-top: 14rem;
+            gap: 1rem;
```
[the remark this answers is not in the document]
Comment (AdelinaMoroaca) [reply style/scss/_header.scss]: Added   gap: 2rem; and  font-size: 2.4rem;

File: script.js [hunk 7]
```diff
@@ -123,9 +123,65 @@ function hideMenuContent() {
...
+
```
[the remark this answers is not in the document]
Comment (AdelinaMoroaca) [reply script.js:133]: when the element is clicked the line color is now green mint 
hovering on the links - need to be fixed

File: style/scss/_header.scss [hunk 8]
```diff
@@ -375,5 +381,64 @@ header {
...
+            }
+
+            & .join-btn {
+                font-size: 2.4rem;
+            }
+        }
+    }
+}
+
+@media (max-width: $phone-size){
```
[the remark this answers is not in the document]
Comment (AdelinaMoroaca) [reply style/scss/_header.scss:426]: new navbar-style for screen sizes between 1024 and 768px added

**Review by talidag — APPROVED** [review]

**Pull request merged**